cypress/integration/add_item.js

```diff
@@ -2,24 +2,37 @@ describe("Add Item To List", function() {
   beforeEach(function() {
     window.localStorage.setItem("uniqueToken", "token1234");
   });
-
   it("Checks that entered items are added to list", function() {
     cy.visit("/add");
     cy.get(".inputField").type("New Cypress Item");
-
     cy.get("#addItemButton").click();
   });
-
   it("Input field accepts input", function() {
     cy.visit("/add");
     cy.get(".inputField")
       .type("watermelon")
       .should("have.value", "watermelon");
   });
 
+
+  // This test was not passing, because of a conflict with our new functionality
+  // that handles duplicate items. I added a function to generate a random item,
+  // instead of "marshmallows" every time, so now the test can work with a new
+  // item on every run - Ethan
   it("Resets input field after addItem button click", function() {
     cy.visit("/add");
-    cy.get(".inputField").type("marshmallows");
+    cy.get(".inputField").type(randomInput());
```

Comment:
This looks good! Better than our solution - we just deleted it 😂 

One recommendation is to move it above all of the test logic though (line 23). In other situations I've seen helper methods like this extracted fully out of the test suite so it's up on line 1, but not sure if that's best here or not. 🤷‍♀

Reply:
That seems like a good idea!